aikau/src/test/resources/testApp/WEB-INF/classes/alfresco/site-webscripts/alfresco/documentlibrary/TagFilters.get.html.ftl

```diff
@@ -0,0 +1 @@
+<@processJsonModel group="share"/>
```

Comment:
Just an FYI... we don't need the group attribute, no need to change - I'm sure there are still lots scattered around in the templates.